<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>关闭(&amp;C)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>发送(&amp;S)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_chat_history">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_inputbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_userlist">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 3px;

background-color: rgba(255,255,255,1);
color: rgb(11, 11, 11);</string>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <row/>
    <row/>
    <column/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string>用户名</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>IP</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_filename">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>连接 (&amp;C)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ip">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>370</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>192.168.208.130</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>9999</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_upload_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_upload_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>20</width>
      <height>541</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_upload_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>User:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_user">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>340</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>user1</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_upload">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>上传</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_accept">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>接收</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filesize">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_inputbox</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_send</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_ip</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_connect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_port</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_connect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_user</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_connect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
